VAMobile/src/store/slices/claimsAndAppealsSlice.ts

```diff
@@ -48,6 +48,7 @@ export type ClaimsAndAppealsState = {
   loadingAppeal: boolean
   loadingSubmitClaimDecision: boolean
   loadingFileUpload: boolean
+  finishedLoadingClaimsAndAppeals: boolean
```

Comment:
Had to add this because the `loadingClaimsAndAppeals` value is initially false, then becomes true, then false again. The initial false value caused the analytics event to be sent immediately on initial render, then sent again after the data finished loading